### ISSUE TITLE
Published Folders should support relative paths

### ISSUE THREAD

**Issue by DataTraveler1** [ironmansoftware/powershell-universal#2445]
### Summary of the new feature / enhancement

Currently with version 3.9.4, it is not possible to specify a relative path for a published folder. For example, if a directory "Folder1" exists in the PowerShell Universal repository directory then the "Path" of the published folder should be configurable as simply "Folder1" instead of C:\Progra~1\Univers~1\etc.

Implementing this would make it practical to maintain a published folder amongst different PSU nodes where the repository location may differ.

###### below figure shows an example of a published folder with a relative path set that currently does not work
![image](https://github.com/ironmansoftware/issues/assets/537287/881df787-4fd1-4747-bc77-f32acca9d436)

**Comment by schubfre** [comment]
Shouldn't this have been fixed with #2374 ?
I have to be honest, I didn't have the time to test this yet, but if it's fixed and works as suggested in my issue you can do something like

```Powershell
$Parameters = @{
	RequestPath = "/documentation/modules/"
	Path = "..\dashboards\ModuleDocumentation\docs"
	Name = "ModuleDocumentation"
}
New-PSUPublishedFolder @Parameters
```

This is how I use relative paths in all my files.

I will test tomorrow if the fix works.

To be honest, I don't see the extra value in leaving out the "..\\" . Contrary this would kind of negate the ability to use folders under ".universal" (for whatever reason one might want to do that)

**Comment by adamdriscoll** [comment]
@DataTraveler1 @schubfre - I just double checked and this is functioning properly. 

The root is set to .universal. To get to a folder in the Repository folder, you will need to move up a directory.
